docs/data/sdk-plugins.md

```diff
@@ -103,18 +103,68 @@ Use an Enrichment Plugin to modify event properties:
               async execute(event: Event): Promise<Event | null> {
                 // ignore events with a certain property
                 if (event.event_properties['ignore'] === true){
-                // returning null will prevent this event from being processed by subsequent plugins
-                return null;
-              }
+                  // returning null will prevent this event from being processed by subsequent plugins
+                  return null;
+                }
 
-              // Allow other events to be processed and sent to destination plugins
-              return event;
+                // Allow other events to be processed and sent to destination plugins
+                return event;
               }
             }
 
             amplitude.add(new FilterEventsPlugin());
             amplitude.init('API_KEY');
             ```
+    
+    ???code-example "Set universal user properties(click to expand)"
+
+        === "JavaScript"
+            ```js
+            import * as amplitude from '@amplitude/analytics-browser';
+    
+            import {PluginType} from '@amplitude/analytics-types';
+    
+            class PropertiesEnrichmentPlugin {
+              name = 'properties-plugin';
+              type = PluginType.ENRICHMENT;
+    
+              async setup(_, amplitude) {
+                if (shouldSetUserProperties) {
+                  const identifyEvent = new amplitude.Identify();
+                  identifyEvent.set("testKey", "testValue");
+                  amplitude.identify(identifyEvent);
+                }
+                return undefined;
+              }
+            }
+    
```

Comment:
Do we need `execute()`?

Reply:
execute is optional I think. https://www.docs.developers.amplitude.com/data/sdks/browser-2/#pluginexecute-optional-for-type-enrichment

Reply:
But I notice that execute is not optional in the order version of the SDK. Thanks for calling that out.  I will add that back.